src/main/java/Menu.java

```diff
@@ -0,0 +1,38 @@
+import com.github.lalyos.jfiglet.FigletFont;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class Menu {
+    private static final int OPTION_DATA_ENTRY = 1;
+    private static final int OPTION_SCHOOL_MANAGEMENT = 2;
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String welcomeMenu = FigletFont.convertOneLine("Iron-School");
+        System.out.println(welcomeMenu);
+        int selectedOption = showPrincipalMenuAndRetrieveOption(sc);
+        System.out.println("Option Selected: " + selectedOption);
+    }
+
+        public static int showPrincipalMenuAndRetrieveOption(Scanner sc) {
+            System.out.println("Select option");
+            System.out.println(OPTION_DATA_ENTRY + " -> Data Entry");
```

Comment:
Here perhaps is better to use an Enum and print its values. Somthing like..
Stream.of(MainMenu.values())
                        .forEach(entry -> System.out.println((entry.ordinal() + 1) + " - " + entry.getDescription()));

And the Enum could be..
public enum MainMenu {
    OPTION_DATA_ENTRY("Data Entry"),
    OPTION_SCHOOL_MANAGEMENT("School Management");

    private final String description;

    MainMenu(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }
}

Reply:
Your advice has been implemented and refactored and test refactored too